patches/MAGETWO-93083__optimize_retrieving_product_attributes__2.2.5.patch

```diff
@@ -0,0 +1,185 @@
+diff --git a/vendor/magento/module-catalog/Model/Product.php b/vendor/magento/module-catalog/Model/Product.php
```

Comment:
Change first line to be same as our other patches with parameters. Also drop index